posts/2021-the-bottom-line.md

```diff
@@ -0,0 +1,177 @@
e](https://github.com/FrickHazard), [Moritz Doll](https://github.com/mcdoll), [Jon Eugster](https://github.com/joneugster), [Moritz Firsching](https://github.com/mo271), [Mark Gerads](https://github.com/Nazgand), [Vladimir Goryachev](https://github.com/SymmetryUnbroken), [Mathieu Guay-Paquet](https://github.com/mguaypaq), [Violeta Hernández](https://github.com/vihdzp), [Christopher Hoskin](https://github.com/mans0954), [Joseph Hua](https://github.com/Jlh18), [María Inés de Frutos-Fernández](https://github.com/mariainesdff), [Ashwin Iyengar](https://github.com/ashwiniyengar), [Matt Kempster](https://github.com/matt-kempster), [Luke Kershaw](https://github.com/l-kershaw), [Huỳnh Trần Khanh](https://github.com/huynhtrankhanh), [Praneeth Kolichala](https://github.com/prakol16), [Alex Kontorovich](https://github.com/AlexKontorovich), [Kalle Kytölä](https://github.com/kkytola), [Julian Külshammer](https://github.com/Julian-Kuelshammer), [Antoine Labelle](https://github.com/antoinelab01), [Ryan Lahfa](https://github.com/RaitoBezarius), [Paul Lezeau](https://github.com/Paul-Lez), [Jireh Loreaux](https://github.com/j-loreaux), [Bernd Losert](https://github.com/berndlosert), [Giacomo Maletto](https://github.com/GiacomoMaletto), [Marc Masdeu](https://github.com/mmasdeu), [Chase Meadors](https://github.com/cemulate), [Yuma Mizuno](https://github.com/yuma-mizuno), [Gabriel Moise](https://github.com/gabrielmoise), [Sebastian Monnet](https://github.com/Sebastian-Monnet), [Hunter Monroe](https://github.com/hmonroe), [Apurva Nakade](https://github.com/apurvnakade), [Anupam Nayak](https://github.com/AaronGreen001), [Peter Nelson](https://github.com/apnelson1), [Luke Nelson](https://github.com/lukenels), [Arthur Paulino](https://github.com/arthurpaulino), [Sorawee Porncharoenwase](https://github.com/sorawee), [Stuart Presnell](https://github.com/stuart-presnell), [Greg Price](https://github.com/gnprice), [Ethan Pronovost](https://github.com/EPronovost), [Jakob von Raumer](https://github.com/javra), [David Renshaw](https://github.com/dwrensha), [Joël Riou](https://github.com/joelriou), [Eric Rodriguez](https://github.com/ericrbg), [Shadman Sakib](https://github.com/shadasali), [Jakob Scholbach](https://github.com/JakobScholbach), [Andrew Souther](https://github.com/asouther4), [Justus Springer](https://github.com/justus-springer), [François Sunatori](https://github.com/frankymacster), [Henry Swanson](https://github.com/HenrySwanson), [Ben Toner](https://github.com/bentoner), [Alain Verberkmoes](https://github.com/verberkm), [Way Yan Win](https://github.com/greysome), [ Winston](https://github.com/winestone), [Ines Wright](https://github.com/ineswright), [Andrew Yang](https://github.com/erdOne), [Yourong Zang](https://github.com/justadzr), [Hanting Zhang](https://github.com/acxxa), [Alex Zhao](https://github.com/cocohearts), [ hallow-world](https://github.com/hallow-world), [ l534zhan](https://github.com/l534zhan), [ thejohncrafter](https://github.com/thejohncrafter)
```

Comment:
```suggestion
Mohamed Al-Fahim<!-- Github account has been deleted  -->, [David Kurniadi Angdinata](https://github.com/Multramate), [Noam Atar](https://github.com/atarnoam), [Mantas Baksys](https://github.com/MantasBaksys), [Itai Bar-Natan](https://github.com/itaibn), [Chris Birkbeck](https://github.com/CBirkbeck), [Manuel Candales](https://github.com/manuelcandales), [Carlos Caralps](https://github.com/carloscaralps), [Robin Carlier](https://github.com/robincarlier), [Antoine Chambert-Loir](https://github.com/AntoineChambert-Loir), [Tian Chen](https://github.com/peakpoint), [Joanna Choules](https://github.com/jchoules), [Iván Sadofschi Costa](https://github.com/isadofschi), [Sara Díaz Real](https://github.com/saradiazr11), [Yaël Dillies](https://github.com/YaelDillies), [Ender Doe](https://github.com/FrickHazard), [Moritz Doll](https://github.com/mcdoll), [Jon Eugster](https://github.com/joneugster), [Moritz Firsching](https://github.com/mo271), [Mark Gerads](https://github.com/Nazgand), [Vladimir Goryachev](https://github.com/SymmetryUnbroken), [Mathieu Guay-Paquet](https://github.com/mguaypaq), [Violeta Hernández](https://github.com/vihdzp), [Christopher Hoskin](https://github.com/mans0954), [Joseph Hua](https://github.com/Jlh18), [María Inés de Frutos-Fernández](https://github.com/mariainesdff), [Ashwin Iyengar](https://github.com/ashwiniyengar), [Matt Kempster](https://github.com/matt-kempster), [Luke Kershaw](https://github.com/l-kershaw), [Huỳnh Trần Khanh](https://github.com/huynhtrankhanh), [Praneeth Kolichala](https://github.com/prakol16), [Alex Kontorovich](https://github.com/AlexKontorovich), [Kalle Kytölä](https://github.com/kkytola), [Julian Külshammer](https://github.com/Julian-Kuelshammer), [Antoine Labelle](https://github.com/antoinelab01), [Ryan Lahfa](https://github.com/RaitoBezarius), [Paul Lezeau](https://github.com/Paul-Lez), [Jireh Loreaux](https://github.com/j-loreaux), [Bernd Losert](https://github.com/berndlosert), [Giacomo Maletto](https://github.com/GiacomoMaletto), [Marc Masdeu](https://github.com/mmasdeu), [Chase Meadors](https://github.com/cemulate), [Yuma Mizuno](https://github.com/yuma-mizuno), [Gabriel Moise](https://github.com/gabrielmoise), [Sebastian Monnet](https://github.com/Sebastian-Monnet), [Hunter Monroe](https://github.com/hmonroe), [Apurva Nakade](https://github.com/apurvnakade), [Anupam Nayak](https://github.com/AaronGreen001), [Peter Nelson](https://github.com/apnelson1), [Luke Nelson](https://github.com/lukenels), [Arthur Paulino](https://github.com/arthurpaulino), [Sorawee Porncharoenwase](https://github.com/sorawee), [Stuart Presnell](https://github.com/stuart-presnell), [Greg Price](https://github.com/gnprice), [Ethan Pronovost](https://github.com/EPronovost), [Jakob von Raumer](https://github.com/javra), [David Renshaw](https://github.com/dwrensha), [Joël Riou](https://github.com/joelriou), [Eric Rodriguez](https://github.com/ericrbg), [Shadman Sakib](https://github.com/shadasali), [Jakob Scholbach](https://github.com/JakobScholbach), [Andrew Souther](https://github.com/asouther4), [Justus Springer](https://github.com/justus-springer), [François Sunatori](https://github.com/frankymacster), [Henry Swanson](https://github.com/HenrySwanson), [Ben Toner](https://github.com/bentoner), [Alain Verberkmoes](https://github.com/verberkm), [Way Yan Win](https://github.com/greysome), [ Winston](https://github.com/winestone), [Ines Wright](https://github.com/ineswright), [Andrew Yang](https://github.com/erdOne), [Yourong Zang](https://github.com/justadzr), [Hanting Zhang](https://github.com/acxxa), [Alex Zhao](https://github.com/cocohearts), [ hallow-world](https://github.com/hallow-world), [Alex Zhang](https://github.com/l534zhan), [ thejohncrafter](https://github.com/thejohncrafter)
```
according to Zulip.

Reply:
(maybe these should be one per line so that that diff makes sense - I added a name for `l534zhan`)

Reply:
In the interest of not de-anonymizing anyone, I only used names that could be mechanically connected to GitHub accounts. (Essentially: name `N` connects to GitHub user `U` if there's a commit in mathlib from `U` using an email that Zulip associates to `N`, or the name on the commit is `N`, or the name on GitHub account `U` is `N`.) I was surprised that this only left 5 anonymous contributors. How clear is the connection for `l534zhan`?

Reply:
> How clear is the connection for l534zhan

Github and zulip use the same email address, and the latter has a name set. I suppose the easiest solution here is just to ping all the anonymous users and ask whether they'd like to provide a name for this blog post.

Reply:
See my second comment on this PR! But this case sounds like a failure of my script, so I'll make the change for `l534zhan`.